workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Editor/Prefabs/PrefabPreprocessor.cs

```diff
@@ -80,11 +82,16 @@ private static bool IsBehaviourEnabledInEditor(Object obj)
 
         private static bool DoesBehaviourNeedConditionalEnabling(Type targetType)
         {
-            return
-                targetType.GetCustomAttribute<WorkerTypeAttribute>() != null ||
-                targetType
-                    .GetFields(BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public)
-                    .Any(field => Attribute.IsDefined(field, typeof(RequireAttribute), false));
+            if (!ConditionalEnableCache.TryGetValue(targetType, out var needsConditionalEnabling))
```

Comment:
Was the lack of this causing a performance problem?

Reply:
This is a continuation of the previous PR #366, where opportunities to improve the neighborhood of the affected code have been discovered.

Reply:
In short: no real problem, it's a driveby.

Reply:
But what is the drive-by fixing? It makes the code harder to read and adds state to and editor only script, which previously had none. So if this isn't something that causes an issue it probably shouldn't be added.

Reply:
Not a fix, an optimization. 

How much faster? I'm unsure, I don't think it's worth a benchmark.

Making the code harder to read: I sort of see what you mean.

I think it's worth shaving off some time from large projects, but can be done at a later phase instead. I'll revert it until it becomes an issue.